hy/contrib/walk.hy

```diff
@@ -223,13 +220,15 @@
               [(= (get form 0) (hy.models.Symbol "require"))
                (ast-compiler.compile form)
                (return)]
-              [True (traverse (mexpand form module ast-compiler))])
+              [(in (get form 0) '[except unpack-mapping])
+               (hy.models.Expression [(get form 0) #* (traverse (cut form 1 None))])]
+              [True (traverse (mexpand form ast-compiler.module ast-compiler :result-ok False))])
         (if (coll? form)
             (traverse form)
             form)))
   (expand form))
 
-(setv _mangled-special (frozenset (map hy.mangle (special))))
+(setv _mangled-core-macs (frozenset (map hy.mangle (hy.extra.reserved.macros))))
```

Comment:
nitpick: might as well call this `_mangled-core-macros` just so it's clear

Reply:
Done.